api/src/main/java/org/pmiops/workbench/ras/RasLinkConstants.java

```diff
@@ -0,0 +1,42 @@
+package org.pmiops.workbench.ras;
+
+import com.google.common.collect.ImmutableSet;
+import java.util.Set;
+
+/** Constants used in to process RAS linkage. */
+public class RasLinkConstants {
+  private RasLinkConstants() {}
+
+  // The RAS url suffix for exchanging token using auth code.
+  static final String TOKEN_URL_SUFFIX = "/auth/oauth/v2/token";
+
+  // The RAS url suffix for initializing authorize request.
+  static final String AUTHORIZE_URL_SUFFIX = "/auth/oauth/v2/authorize";
+
+  // The RAS url suffix for fetching user info, i.e. GA4GH Passport.
+  static final String USER_INFO_URL_SUFFIX = "/openid/connect/v1.1/userinfo";
+
+  // The GCP bucket name that stores RAS secret
+  static final String RAS_SECRET_BUCKET_NAME = "ras-client-secret.txt";
+
+  // The OIDC acr (Authentication Context Class Reference) claim name in JWT
+  static final String ACR_CLAIM = "acr";
+
+  // The Id Token field name from RAS's TokenResponse
+  static final String Id_TOKEN_FIELD_NAME = "id_token";
+  // The login.gov's PREFERRED_USERNAME field from RAS UserInfo endpoint
+  static final String PREFERRED_USERNAME_FIELD_NAME = "preferred_username";
+
+  // The login.gov's email field which we choose as login.gov username.
+  static final String EMAIl_FIELD_NAME = "email";
+
+  // The identifier that indicate users login using login.gov account.
+  static final String LOGIN_GOV_IDENTIFIER_LOWER_CASE = "@login.gov";
+
+  // The prefix in ACR claim that we can use to identify user IAL status.
+  // ACR_CLAIM_IAL_PREFIX + {1/2} is the user's IAL tier.
+  static final String ACR_CLAIM_IAL_PREFIX = "stsstg.nih.gov/assurance/ial/";
```

Comment:
Hm, do we actually care about the prefix? Might be simpler to just have a single string constant for the whole thing (ACR_CLAIM_IAL2), we don't really use IAL1 anyways

Reply:
I was thinking invalid acr vs IA1 different cases, and we want to treat them differently in UI. But I guess we just care 2 vs not 2.